internal/cli/command/login/oidc.go

```diff
@@ -0,0 +1,343 @@
+	default:
+		http.Error(w, fmt.Sprintf("method not implemented: %s", r.Method), http.StatusBadRequest)
+		return
+	}
+
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to get token: %v", err), http.StatusInternalServerError)
+		return
+	}
+
+	rawIDToken, ok := token.Extra("id_token").(string)
+	if !ok {
+		http.Error(w, "no id_token in token response", http.StatusInternalServerError)
+		return
+	}
+
+	idToken, err := a.verifier.Verify(r.Context(), rawIDToken)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Failed to verify ID token: %v", err), http.StatusInternalServerError)
+		return
+	}
+
+	var claims json.RawMessage
+	idToken.Claims(&claims)
+
+	buff := new(bytes.Buffer)
+	json.Indent(buff, []byte(claims), "", "  ")
+
+	err = a.requestTokenFromPipeline(rawIDToken)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to request Pipeline token: %v", err), http.StatusInternalServerError)
+		return
+	}
+
+	renderClosingTemplate(w)
+
+	log.Info("successfully logged in")
+}
+
+func (a *app) requestTokenFromPipeline(rawIDToken string) error {
+	pipelineURL, err := url.Parse(a.pipelineBasePath)
+	if err != nil {
+		return err
+	}
+
+	pipelineURL.Path = "/auth/dex/callback"
+
+	reqBody := bytes.NewBuffer(nil)
+	writer := multipart.NewWriter(reqBody)
+	writer.WriteField("id_token", rawIDToken)
+	writer.Close()
+
+	client := &http.Client{
+		CheckRedirect: func(req *http.Request, via []*http.Request) error {
+			return http.ErrUseLastResponse
+		},
+	}
+
+	resp, err := client.Post(pipelineURL.String(), writer.FormDataContentType(), reqBody)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	if resp.StatusCode < 200 || resp.StatusCode >= 400 {
+		body, _ := ioutil.ReadAll(resp.Body)
+		return fmt.Errorf("request returned: %s", string(body))
+	}
+
+	for _, cookie := range resp.Cookies() {
+		if cookie.Name == "user_sess" {
+			a.banzaiCli.Context().SetToken(cookie.Value)
+			return nil
+		}
+	}
+
+	return fmt.Errorf("failed to find user_sess cookie in Pipeline response")
+}
+
+func open(url string) error {
+
+	var cmd string
+	var args []string
+
+	switch runtime.GOOS {
+	case "windows":
+		cmd = "cmd"
+		args = []string{"/c", "start"}
+	case "darwin":
+		cmd = "open"
+	default: // "linux", "freebsd", "openbsd", "netbsd"
+		cmd = "xdg-open"
+	}
+	args = append(args, url)
+	return exec.Command(cmd, args...).Start()
+}
+
+func (a *app) waitShutdown() {
+	irqSig := make(chan os.Signal, 1)
+	signal.Notify(irqSig, syscall.SIGINT, syscall.SIGTERM)
+
+	// Wait interrupt or shutdown request through /shutdown
+	select {
+	case sig := <-irqSig:
+		log.Printf("Shutdown request (signal: %v)", sig)
+		os.Exit(0)
+	case <-a.shutdownChan:
+		os.Exit(0)
+	}
+}
+
+func renderClosingTemplate(w io.Writer) {
+	w.Write([]byte(closingTemplate))
+}
+
+const closingTemplate = `
+<!DOCTYPE html>
+<html>
+<head>
+<script type="text/javascript">
+	setTimeout(function () { window.location.href = "https://beta.banzaicloud.io/docs/"; }, 5000);
```

Comment:
```suggestion
	setTimeout(function () { window.location.href = "https://banzaicloud.com/docs/"; }, 5000);
```

(we will use this on the long term, and works already)